Mudanças em relação ao código original

cvrsebastiany opened the pull request:
- Exclusão da função insereNodo() na Pilha, por ser muito parecida com a função insereNodoInicio(), e inútil nesse contexto
- Correção da chamada de função no Main, de insereNodo() para insereNodoInicio()
- Melhoras no menu
- Mudanças na formatação e diagramação do código, para melhorar a leitura

cvrsebastiany committed:
Update main

Melhora do menu, correção da chamada do método insereNodoInicio e outras mudanças menores

cvrsebastiany committed:
Update main

cvrsebastiany committed:
Update Pilha

Mudanças gerais de espaçamento e remoção da função insereNodo, por ser apenas uma duplicata com ação equivocada da função insereNodoInicio

cvrsebastiany committed:
Update Pilha

Adição da package

isadora-vramos commented:
A exclusão do método que era redundante facilita bastante o entendimento de quem acessar esse código no futuro. Uma sugestão é adicionar uma mensagem para o usuário quando ele solicita uma opção diferente das possíveis, pedindo para informar novamente.

cvrsebastiany committed:
Update main

Seguindo a sugestão da colega Isadora, foi adicionado um default ao fim do switch case,
